pylib/mozhginfo/tests/test_pushlog_client.py

```diff
@@ -1,3 +1,7 @@
+"""
+these tests will no longer workable after bug 1228358 been landed,
+ we will totally rewrite them in bug 1227531.
+"""
```

Comment:
This is inaccurate. bug 1227531 intend to drop using Mocking (as we currently use in these tests).
Until we start working on bug 1227531 we should not be loosing any test coverage.